### PR TITLE
V22F-815 Fix key controls in all menus

### PR DESCRIPTION
**Pull request by shoekzema**
Added default key controls to the options menus
Added 'Esc' key to go back to Database Connection Menu and Researcher Menu
Fixed the press Enter in EndOfExperiment Screen
Changed the order the key controls go in (except for the keybindings in controls, cause you either have to hardcode it (which is not maintainable) or rewrite like 50 files)
Swapped the positions of 'Yes, quit' and 'No, go back' in the Quit screen.

### CODE REVIEW

File: speed-dreams/source-2.2.3/data/data/menu/controlconfigmenu.xml [hunk 1]
```diff
@@ -721,7 +721,7 @@
 <!-- Buttons -->
 	<section name="cancel">
 		<attstr name="type" val="text button"/>
-		<attstr name="show box" val="no"/>
+		<attstr name="show box" val="yes"/>
```
Comment (larsdekwant) [inline speed-dreams/source-2.2.3/data/data/menu/controlconfigmenu.xml:724]:
```suggestion
		<attstr name="show box" val="no"/>
```

File: speed-dreams/source-2.2.3/data/data/menu/controlconfigmenu.xml [hunk 2]
```diff
@@ -743,7 +743,7 @@
 
 	<section name="calibrate">
 		<attstr name="type" val="text button"/>
-		<attstr name="show box" val="no"/>
+		<attstr name="show box" val="yes"/>
```
Comment (larsdekwant) [inline speed-dreams/source-2.2.3/data/data/menu/controlconfigmenu.xml:746]:
```suggestion
		<attstr name="show box" val="no"/>
```

File: speed-dreams/source-2.2.3/data/data/menu/controlconfigmenu.xml [hunk 3]
```diff
@@ -765,7 +765,7 @@
 
 	<section name="save">
 		<attstr name="type" val="text button"/>
-		<attstr name="show box" val="no"/>
+		<attstr name="show box" val="yes"/>
```
Comment (larsdekwant) [inline speed-dreams/source-2.2.3/data/data/menu/controlconfigmenu.xml:768]:
```suggestion
		<attstr name="show box" val="no"/>
```

**Review by larsdekwant — CHANGES_REQUESTED** [review]
- Order on controlconfig menu is incorrect (skips certain options, then goes back to them later)
- Apply, Calibrate and Canel button don't need to show box on controlconfig menu, they are already highlighted.
- Forcefeedback menu order incorrect.

**Review by suppergerrie2 — CHANGES_REQUESTED** [review]
On the control menu the order is wrong:
Left steer => Right Steer => throttle => brake => (E brake => speed limiter) => clutch => abs => tcs => [speed limiter] => light => (Left glance => right glance) => reverse gear => neutral gear => Down shift => up shift => [e brake] => [left glance] => [right glance] => intervention toggle on/off

options in () should be there but arent, options in [] are there but should be somewhere else 

On the database connection screen:
cant type in user or password screen when selecting it

**Comment by shoekzema** [comment]
I will not (not worth the time) fix the orders in the FF-menu and the controls config menu, because they are made by looping through the buttons based on the order of a struct type that is used in like 50 other places.

I won't state that the GuiOrder changed, because the order only matters with defaultkeysadd, which I did state was changed.

**Review by suppergerrie2 — APPROVED** [review]

**Review by larsdekwant — APPROVED** [review]

**Pull request merged**